server/cmwell-data-tools/src/main/scala/cmwell/tools/data/downloader/consumer/Downloader.scala

```diff
@@ -471,56 +472,59 @@ class Downloader(baseUrl: String,
       val job = Flow[(Seq[Path], State)]
         .map { case (paths, state) => paths -> Some(state) }
         .via(Retry.retryHttp(timeout, parallelism, baseUrl)(createRequest)) // fetch data from paths
+        .map {
```

Comment:
Decompressing here is too late. The body is read also in the retryHttp and won't be parsed correctly.